vpr/src/route/rr_graph_writer.cpp

```diff
@@ -38,11 +38,10 @@ void write_rr_graph(const char* file_name) {
         &device_ctx.rr_nodes,
         &device_ctx.rr_graph_builder,
         &device_ctx.rr_graph,
-        &device_ctx.rr_switch_inf,
+        &device_ctx.rr_graph_builder.rr_switch_inf(),
```

Comment:
This line can be safely removed if you moved ``rr_switch_inf`` from ``DeviceContext`` to ``RRGraphBuilder``. Just like what you have done for ``rr_segments``